op-e2e/celo/run_all_tests.sh

```diff
@@ -50,7 +51,7 @@ then
 	echo All tests succeeded!
 else
 	tput setaf 1 || true
-	echo $failures/$tests failed.
+	echo "$failures"/"$tests" failed.
```

Comment:
I usually prefer quoting everything at once:
```suggestion
	echo "$failures/$tests" failed.
```
But that's quite subjective.